sig-cluster-lifecycle/charter.md

```diff
@@ -0,0 +1,65 @@
+# SIG Cluster Lifecycle Charter
+
+## Scope
+
+SIG Cluster Lifecycle’s objective is to improve the deployment, configuration management, and upgrade story for Kubernetes clusters and relevant infrastructure.
+
+### In scope
+
+The following topics fall under ownership of this SIG:
+
+- Improving the Kubernetes UX for cluster administration
+- Kubernetes cluster lifecycle related features and issues
+- Tools that assist in bootstrapping, mutating, and upgrading Kubernetes control plane components. 
+- Kubernetes tools used to manage infrastructure related to provisioning and upgrading nodes
+- Tools that assist in management of configuration of Kubernetes components
+
+#### Code, Binaries and Services
+
+- Everything that falls in the scope of the SIG.
+- Tools that are provider specific implementation for infrastructure management.
+- Core add-ons that are required for cluster bootstrapping.
```

Comment:
Also, do we need to find owners for the other addons in k/k/cluster?

Reply:
Yes -- other addons need owners. Many (most?) already have them: [calico policy controller](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/calico-policy-controller/OWNERS), [cluster loadbalancing](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/cluster-loadbalancing/OWNERS), [cluster monitoring](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/cluster-monitoring/OWNERS), [dashboard](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dashboard/OWNERS), [dns horizontal autoscaler](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns-horizontal-autoscaler/OWNERS), [dns](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/OWNERS) -- basically every addon I looked at until that point.

Reply:
> Self-hosted cluster components (aka "add ons")

One way we've been trying to think about this is that they are "things" (applications / programs / something) that are managed by the cluster admin rather than by cluster users. They have historically been tied to the lifecycle of the cluster, but I think that is because that is currently the primary touchpoint for the cluster admin persona rather than because they need to be managed that way. 

/cc @justinsb @johnsonj

Reply:
@roberthbailey re the last comment.